control-plane/rest/openapi-specs/v0_api_spec.yaml

```
@@ -2705,11 +2705,15 @@ components:
           type: integer
           format: int32
           minimum: 0
+        encrypted:
+          description: flag indicating whether or not the volume replicas are to be placed on encrypted pool
```

Do we want to be so specific on the description?
Ex: what happens if we introduce encryption on replicas in the future?

```suggestion
        encrypted:
          description: flag indicating whether or not the volume replicas are to be placed on encrypted pool
```

---

Right, how about `flag to enable data encryption` ?

---

Yeah I think that's enough

---

Done